Empire/Empire.cpp

```diff
@@ -976,7 +977,7 @@ void Empire::SetResourceStockpile(ResourceType resource_type, float stockpile) {
 }
 
 void Empire::PlaceTechInQueue(const std::string& name, int pos/* = -1*/) {
-    if (name.empty() || TechResearched(name) || m_techs.count(name))
+    if (name.empty() || TechResearched(name) || m_techs.count(name) || m_new_techs.count(name))
```

Comment:
meaning is unclear and `m_new_techs` lacks context. Please explain with a comment.